### DISCUSSION TITLE
NUC7CJYH

### DISCUSSION THREAD

**Discussion by tommiefr** [nicholas-opensource/OpenWrt-Autobuild#38] [Ideas]
Can you please add support for this NUC? 
I want to replace my R2S for this NUC.
Beside this, can you add the driver for RTL8153/RTL8152? I use an UE300 adapter for lan.

**Comment by nicholas-opensource** [comment]
https://github.com/nicholas-opensource/OpenWrt-Autobuild/commit/04f60682cf9b542756547a79ddb658e32ad13cef done.
I have supported nuc with x86 firmware, and RTL8111 driver is already included (kmod-r8169)
Actually I'm an Intel nuc user too.

**Reply by tommiefr** [reply]
Great, so RTL8111 driver is also usable for UE300, I saw kmod-usb-net-rtl8152 is needed.
Can you add this kmod as well?
Still searching for a manual to install.
Just burn this to disk and boot nuc?
So the 2nd nic is recognized directly?

**Reply by nicholas-opensource** [reply]
I've added rtl8111 and rtl8152, the NICs you mention have drivers built in already! combine-efi.img is better

**Reply by tommiefr** [reply]
Superb! Thanks!
What do you mean with combine-efi.img?
In the X86 zip I see only one image: openwrt-x86-64-generic-squashfs-combined.img
Or do I need first to install the official built and afterwards this one?

**Reply by nicholas-opensource** [reply]
Didn't release yet, you can download in the latest action run. https://github.com/nicholas-opensource/OpenWrt-Autobuild/actions/runs/6112477989
Or waiting the next release, v23.05.0 stable release may be released in October

**Reply by tommiefr** [reply]
Thank you for your quick reply!
Will use that one!

**Reply by tommiefr** [reply]
Just had time to install the latest run. This one is working great! Thanks for your support!

**Comment by tommiefr** [comment]
I cannot install Docker on the x86 version. Docker-ce is not in the list. Is it possible to add this complete package aa well in the next built? 
I want to install home assistant on a Docker in openwrt

**Reply by nicholas-opensource** [reply]
Installing docker on openwrt is not recommended as it will cause abnormal UDP traffic on the bridge device

**Reply by nicholas-opensource** [reply]
I recommend you to install Proxmox VE, a Linux virtual machine system

**Comment by tommiefr** [comment]
ok, decided to add a 2nd NUC for Home Assistant. Configured Openwrt and want to install SQM (luci-app-sqm), but in Luci I get this error:
Unable to execute opkg install command: Error: XHR request aborted by browser

Tried to install it on Putty, but it completely hangs on: configuring kmod-nf-ipt.
Any idea what causes this?

**Reply by nicholas-opensource** [reply]
sqm is outdated software, the current firmware does not support sqm's dependency on iptables (Openwrt v23.05 already defaults to nftables)
